app/models/student.rb

```diff
@@ -13,7 +13,10 @@ class Student < ApplicationRecord
 
   validates :first_name, :last_name, :grad_year, :osis, :email, :phone,
             presence: true, allow_blank: false
+
   validates :parents, presence: true
+
+  validates :osis, length: { is: 9 }
```

Comment:
May also want to include `numericality: { greater_than: 0 }` to make sure we don't get negative OSISes.